rollup.config.ts

```diff
@@ -144,4 +146,25 @@ const buildNative = {
   ],
 }
 
-export default [buildNode, buildNative, buildEsm, buildUdm]
+/**
+ * Configuration for the iife build
+ */
+const buildIife = {
+  input: 'src/index.ts',
+  output: {
+    file: 'lib/iife/index.js',
+    name: 'MockServiceWorker',
+    format: 'iife',
+    esModule: false,
+  },
+  plugins: [
+    ,
```

Comment:
```suggestion
```

Reply:
Fixed, sorry about that!

You can see the file I used to test here: https://gist.github.com/arnaudNYC/a1491c87c86ba38a948482494f04878b

```html
<html>

  <head>
    <script src="./mockServiceWorker.js"></script>
    <script src="./index.js"></script>
  </head>

  <body>
    <script type="text/javascript">
      var rest = MockServiceWorker.rest;
      var setupWorker = MockServiceWorker.setupWorker;

      var handlers = [
        rest.get('/user', function (req, res, ctx) {
          return res(
            ctx.status(200),
            ctx.json({
              username: 'admin',
            }),
          )
        }),
      ];

      var worker = setupWorker.apply(void 0, handlers);
      worker.start();

      fetch('/user')
        .then(function (resp) {
          return resp.json();
        })
        .then(function (data) {
          console.log('Data:', data)
        });
    </script>
  </body>

</html>
```

![Screen Shot 2020-12-15 at 9 30 46 AM](https://user-images.githubusercontent.com/7407917/102228275-84550100-3eb8-11eb-83ad-5355263fab3d.png)